lib/elixir/lib/system.ex

```diff
@@ -5,6 +5,37 @@ defmodule System do
   with the VM or the host system.
   """
 
+  @typedoc """
+  The time unit that can be passed to functions like `System.system_time/1`
+  and similar.
+
+  The `:seconds`, `:milli_seconds`, `:micro_seconds`, and `:nano_seconds` time
+  units are self-explanatory: the return value of the functions that accept a
+  time unit will be in the given time unit.
+
+  `:native` is the time unit used by the Erlang runtime system. It's determined
+  when the runtime starts and stays the same until the runtime is stopped. To
+  determine what the `:native` unit amounts to in a system, you can call
+  `System.convert_time_unit/1` to convert 1 second to the `:native` time unit
+  (i.e., `System.convert_time_unit(1, :seconds, :native)`.
+
+  A time unit can also be a strictly positive integer. In this case, it
+  represents the "parts per second": the time will be returned in
+  `1 / parts_per_second` seconds. For example, the `:milli_seconds` time unit
+  can be simulated by using `1000` as the time unit (as the time will be
+  returned in 1/1000 seconds - milliseconds).
+
+  See also the [documentation on the `time_unit` Erlang
+  type](http://erlang.org/doc/man/erlang.html#type-time_unit).
+  """
+  @type time_unit ::
+    :seconds
+    | :milli_seconds
+    | :micro_seconds
+    | :nano_seconds
```

Comment:
Oh.. I see those are defined in Erlang http://erlang.org/doc/man/erlang.html#type-time_unit